bbb-graphql-server/bbb_schema.sql

```diff
@@ -45,7 +45,9 @@ create table "meeting" (
     "endWhenNoModeratorDelayInMinutes" integer,
 	"endedAt" timestamp with time zone,
 	"endedReasonCode" varchar(200),
-	"endedBy" varchar(50)
+	"endedBy" varchar(50),
+    "screenShareBroadcastAllowedFor" varchar(50),
+    "viewerScreenShareViewAllowedFor" varchar(50)
```

Comment:
In my view, the new columns would be a better fit in the `meeting_usersPolicies` table.